README.md

```diff
@@ -33,9 +33,9 @@ curl https://www.openssl.org/source/openssl-1.0.2r.tar.gz | tar xfz - && cd open
 ###FOP, used to generate docs.
 mkdir /opt/fop
 cd /opt/fop
-wget https://dlcdn.apache.org/xmlgraphics/fop/binaries/fop-2.9-bin.tar.gz
-tar -zxvf fop-2.9-bin.tar.gz
-ln -s  /opt/fop/fop-2.9/fop/fop /usr/bin/fop
+wget https://dlcdn.apache.org/xmlgraphics/fop/binaries/fop-2.10-bin.tar.gz
```

Comment:
as this is a version change, and I expect more in the future, we should later dynamically pull the latest version of FOP, or build a supported version of FOP from source.